src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -391,6 +392,7 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
 
   return (
     <div css={styles.pageWrapper}>
+      <LiteRedirect />
```

Comment:
Is it worth trying this as high up the component tree as possible? Or does it need restricted to Article pages for now?